examples/advanced_examples/plot_mne_features_pipeline.py

```diff
@@ -0,0 +1,97 @@
+"""
+==========================
+Pipelines using the mne-features library
+==========================
+
+This example shows how to evaluate a pipeline constructed using the
+mne-features library
```

Comment:
You also need to mention that it is necessary to install mne-features outside of moabb, it tells a mini-story telling... moabb is really cool, and we have easy interconnectivity with any library that has a sklearn pipeline.